packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -73,6 +74,10 @@ export type DatePickerProps = {
    * A descriptive message for the 'status' states.
    */
   validationMessage?: DateInputFieldProps["validationMessage"] | undefined
+  /**
+   * Configure the floatingUI options that controls the CalendarPopover
+   */
+  floatingOptions?: Partial<UseFloatingOptions>
```

Comment:
I don't think we should be exposing this? What might be the use case for it?

Reply:
Main case I can think of would be for the `strategy` (fixed or absolute) or the preference on the `placement` of the popover. I'd say its not essential but could be a nice to have

Reply:
Axed this prop so should be good to go @HeartSquared